common/src/main/java/uk/co/harieo/forceop/common/DataConverter.java

```diff
@@ -58,8 +58,8 @@ public static String convertBinaryToHex(byte[] bytes) {
 	 * @return the byte
 	 */
 	private static byte decodeHexChar(char hexChar) {
-		if (hexChar >= 65 && hexChar <= 70) {
-			return (byte) (hexChar - 65 + 10); // -65 brings the letter on a scale from 0, +10 because 0-9 are numbers
+		if (hexChar >= 'a' && hexChar <= 'f') {
```

Comment:
ok apply this change throughout the function though ahaha

ie `return (byte) (hexChar - 'a' + 10)`, `else if (hexChar >= '0' && hexChar <= '9')`.

also making this work with both uppercase and lowercase hex strings would be good

Reply:
I've made it convert to lower case just for the method, that way either case would work